core/src/main/scala/org/apache/spark/storage/BlockManagerMaster.scala

```diff
@@ -202,6 +202,14 @@ class BlockManagerMaster(
     Await.result(future, timeout)
   }
 
+  /**
+   * Find out if the executor has cached blocks. This method checks only if the executor holds
+   * cached blocks, which are not broadcast blocks.
```

Comment:
well, technically broadcast blocks can be cached blocks. Maybe it's more accurate to just say "This method does not consider broadcast blocks because those are not reported to the master."